src/roiextractors/extractors/nwbextractors/nwbextractors.py

```diff
@@ -328,9 +328,11 @@ def get_rejected_list(self):
                 return rej_list
 
     def get_images_dict(self):
-        if self._segmentation_images is None:
-            return super().get_images_dict()
-        images_dict = {image_name: image_data[:].T for image_name, image_data in self._segmentation_images.items()}
+        images_dict = super().get_images_dict()
+        if self._segmentation_images is not None:
+            images_dict.update(
+                (image_name, image_data[:].T) for image_name, image_data in self._segmentation_images.items()
+            )
```

Comment:
@h-mayorquin 
This seems like finally fixed the gin tests in catalystneuro/neuroconv/#87. So why is that? (thinking out loud)
Our segmentation extractor writes segmentation data with only `correlation` or images with some other name to an nwbfile. Then, if we wanted to test whether data was written correctly we are reading this nwbfile in with `NwbSegmentationExtractor`. Why override get_images_dict() in the first place? This was necessary to allow saving images that are not named "correlation" or "mean" images -- as it is defined in `SegmentationExtractor.get_images_dict()`. Without the super() class call if a segmentation extractor only wrote "correlation" image then the comparison would fail because its get_images_dict() would return also "mean" image that is None --> we should also have this `NwbSegmentationExtractor`. 

The hotfix for `ExtractSegmentationExtractor` will be done in a separate PR, I just wanted to see if I add the super() call there then the ExtractSegmentationInterface would pass also this test.

Reply:
I'd agree - neither ROIExtractors base nor the NWB extractor should expect 'mean' and 'correlation' to be the only (or any) specified summary images - different segmentation methods seem to have a wide variety of statistical methods for displaying/summarizing the input.

Reply:
Yes, I'm just pointing out that the way I managed to solve this issue is if there are other images other than "correlation" or "mean" then the super() class call will be necessary when overriding the parent class SegmentationExtractor's `get_images_dict()` method. This was the best solution I come up today to make all previous tests pass with this change (of having other images in segmentation).